### PR TITLE
Use github.com/microsoft casing

### PR DESCRIPTION
**Pull request by chwarr**
The "Microsoft" organization on GitHub's canonical URL is
https://github.com/microsoft with a lowercase 'm'.

### CODE REVIEW

**Comment by chwarr** [comment]
Should I bump the version of these files in this PR, or do they get bumped whenever "a release" is cut?

File: projections/dnfmit/SECURITY.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
```
Comment (chwarr) [inline projections/dnfmit/SECURITY.md:2]: Should I bump the version of these files in this PR, or do they get bumped whenever "a release" is cut?

Comment (jeffwilcox) [reply projections/dnfmit/SECURITY.md:2]: If you could bump the comment at the top of the Markdown files, that'd help!

Comment (jeffwilcox) [inline projections/dnfmit/SECURITY.md:2]: pls bump and thanks for the reminder on that!

**Review by jeffwilcox — CHANGES_REQUESTED** [review]

**Review by jeffwilcox — APPROVED** [review]

**Comment by jeffwilcox** [comment]
Thanks :shipit:

**Pull request merged**